packages/surveys/src/components/general/AddedImage.tsx

```diff
@@ -0,0 +1,13 @@
+interface AddedImageProps {
+  imgUrl: string;
+  altText?: string;
+}
+
+export default function AddedImage({ imgUrl, altText = "Image" }: AddedImageProps) {
```

Comment:
I think we can improve the name here as `AddedImage` is not really saying what it is. Either just `Image` or `QuestionImage` if that's what it is?